Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -77,6 +77,25 @@ extension PinsStore.PinState: Codable {
     }
 }
 
+#if compiler(>=6.0)
+
+extension PinsStore.PinState: @retroactive Hashable {
```

Comment:
`@retroactive` is not supported on Swift 5, so these implementations are just copied.